pom.xml

```diff
@@ -3000,6 +3000,7 @@
                 reduce the cost of migration in subsequent versions.
               -->
               <arg>-Wconf:cat=deprecation&amp;msg=it will become a keyword in Scala 3:e</arg>
+              <arg>-Wconf:cat=deprecation&amp;msg=Unicode escapes in triple quoted strings are deprecated:e</arg>
```

Comment:
@panbingkun please revert the change of `SparkBuild.scala` and `pom.xml`

Reply:
Okay